rust/chains/hyperlane-ethereum/src/mailbox.rs

```diff
@@ -312,7 +316,8 @@ where
             message: RawHyperlaneMessage::from(message).to_vec().into(),
             metadata: metadata.to_vec().into(),
         };
-        process_call.encode()
+
+        AbiEncode::encode(process_call)
```

Comment:
jw why this change?
like even if it's just a stylistic choice, for my own learning am curious why